addons/survey/views/survey_templates_user_input_session.xml

```diff
@@ -204,8 +204,8 @@
                     </div>
                     <!-- We keep "18rem" of space to display the points / nickname.
                     Then, the length of the bar is a percentage of the attendee's score compared to the max_score. -->
-                    <t t-set="width_ratio" t-value="round(round(score['scoring_total']) / round(max_score), 3)"/>
-                    <t t-set="width_ratio_question" t-value="str(round(round(score.get('question_score', 0)) / round(score.get('max_question_score', 1)), 3))"/>
+                    <t t-set="width_ratio" t-value="round(round(score['scoring_total']) / (round(max_score, 3) or 1), 3)"/>
```

Comment:
So here you first round to 3 decimals to be more flexible and allow 0.0005 and above, is that right? Others will be considered as "1" and give the tiniest width_ratio known to man ^^ (but should never happen probably).